packages/connectivity/src/scp-cf/destination/destination-service.spec.ts

```diff
@@ -175,6 +211,36 @@ describe('destination service', () => {
       });
     });
 
+    it('returns if some destinations are broken', async () => {
```

Comment:
```suggestion
    it('returns if some destinations are broken - subaccount destinations', async () => {
```

Reply:
I found it hard to see how the tests are different - maybe the different name makes it more obvious